### ISSUE TITLE
500 Server Error

### ISSUE THREAD

**Issue by hj9420** [rkizy617/HRTechnologies#4]
We are getting a 500 server error when running application locally

![image](https://user-images.githubusercontent.com/91652207/139965201-4527c3ce-9557-4072-9452-c8cf15f19dfe.png)

**Comment by hj9420** [comment]
```sh

[2021-11-02 22:59:41] production.ERROR: No application encryption key has been specified. {"exception":"[object] (Illuminate\\Encryption\\MissingAppKeyException(code: 0): No application encryption key has been specified. at /app/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php:101)
[stacktrace]
#0 /app/vendor/laravel/framework/src/Illuminate/Support/helpers.php(263): Illuminate\\Encryption\\EncryptionServiceProvider->Illuminate\\Encryption\\{closure}()
#1 /app/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php(103): tap()
#2 /app/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php(82): Illuminate\\Encryption\\EncryptionServiceProvider->key()
#3 /app/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php(34): Illuminate\\Encryption\\EncryptionServiceProvider->parseKey()
#4 /app/vendor/laravel/framework/src/Illuminate/Container/Container.php(873): Illuminate\\Encryption\\EncryptionServiceProvider->Illuminate\\Encryption\\{closure}()
#5 /app/vendor/laravel/framework/src/Illuminate/Container/Container.php(758): Illuminate\\Container\\Container->build()
#6 /app/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(841): Illuminate\\Container\\Container->resolve()
#7 /app/vendor/laravel/framework/src/Illuminate/Container/Container.php(694): Illuminate\\Foundation\\Application->resolve()
#8 /app/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(826): Illuminate\\Container\\Container->make()
#9 /app/vendor/laravel/framework/src/Illuminate/Container/Container.php(1031): Illuminate\\Foundation\\Application->make()
#10 /app/vendor/laravel/framework/src/Illuminate/Container/Container.php(951): Illuminate\\Container\\Container->resolveClass()
#11 /app/vendor/laravel/framework/src/Illuminate/Container/Container.php(912): Illuminate\\Container\\Container->resolveDependencies()
#12 /app/vendor/laravel/framework/src/Illuminate/Container/Container.php(758): Illuminate\\Container\\Container->build()
#13 /app/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(841): Illuminate\\Container\\Container->resolve()
#14 /app/vendor/laravel/framework/src/Illuminate/Container/Container.php(694): Illuminate\\Foundation\\Application->resolve()
#15 /app/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(826): Illuminate\\Container\\Container->make()
#16 /app/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(206): Illuminate\\Foundation\\Application->make()
#17 /app/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(180): Illuminate\\Foundation\\Http\\Kernel->terminateMiddleware()
#18 /app/public/index.php(55): Illuminate\\Foundation\\Http\\Kernel->ter
```
**(Illuminate\\Encryption\\MissingAppKeyException(code: 0): No application encryption key has been specified. at** 
Also I have noticed there is no environment file for and env.example  , copied this to .env and pushed it to a branch.
Also I made adjustments to the environment variables  to match the docker environment 
 
On the container is ran 
```sh
$ php artisan generate:key
```
![image](https://user-images.githubusercontent.com/91652207/139965585-b737ac43-9b19-488a-9f18-1daf3af9a022.png)

This is a one time thing but if it happens a gain spin up the docker containers and run the command mentioned after ssh into the docker container for the frontend.

**Comment by rkizy617** [comment]
I will look into this, however, I am not sure what you mean by "Also I have noticed there is no environment file for and env.example , copied this to .env and pushed it to a branch. Also, I made adjustments to the environment variables to match the docker environment". **What is an environment file and env.example.**
Also, I am still getting the 500 | Server Error on my end. I don't understand what you mean by "run the command after ssh into the docker container".
Finally with these fixed changes, should I merge the new branch into the main branch
